tests/resources/test_group.py

```diff
@@ -1,15 +1,22 @@
-from tests.conftest import JiraTestCase
+from tests.conftest import JiraTestCase, allow_on_cloud
 
 
+@allow_on_cloud
 class GroupsTest(JiraTestCase):
+    def setUp(self) -> None:
+        JiraTestCase.setUp(self)
```

Comment:
is there a reason why not to do 

`super().setUp(self)`

Reply:
No reason, happy to change!